### PR TITLE
topwar.ru

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

**Comment by jeremyn** [comment]
Thanks.

- [ ] https://forum.topwar.ru and https://cdn.topwar.ru work for me, you can uncomment them
- [ ] Note in a top comment that https://www.topwar.ru , https://mt.topwar.ru , and https://wf.topwar.ru have certificate mismatches

**Comment by galeksandrp** [comment]
Added.

**Comment by jeremyn** [comment]
https://forum.topwar.ru/ is still valid HTTPS, even though it currently redirects immediately to http.

File: src/chrome/content/rules/topwar.ru.xml [hunk 1]
```diff
@@ -1,8 +1,17 @@
+<!--
+www.topwar.ru ¹
+forum.topwar.ru ⁶
```
Comment (galeksandrp) [inline src/chrome/content/rules/topwar.ru.xml]: Yes.

**Comment by jeremyn** [comment]
If the HTTPS is valid then why not include it as a target?

**Comment by galeksandrp** [comment]
Redirect loop? Even if it mitigated after ~3loops by HTTSE, it is increase page load time.

**Comment by jeremyn** [comment]
Oops, you're right. I didn't think about it looping. It should stay commented.

**Comment by jeremyn** [comment]
Thanks, merged.

**Pull request merged**